### DISCUSSION TITLE
Double lsp text

### DISCUSSION THREAD

**Discussion by navxio** [LazyVim/LazyVim#1321] [General]
<img width="896" alt="image" src="https://github.com/LazyVim/LazyVim/assets/9297865/0b1fd324-204e-4b0c-8658-7948d692de7f">

I've only enabled the typescript plugin and don't set up any other javascript/react server..

**Comment by dpetka2001** [comment]
Maybe it's coming from `lsp_lines`? Can you disable that temporarily and observe its behavior?

**Comment by navxio** [comment]
@dpetka2001 Removed lsp_lines but still face the same problem

<img width="1344" alt="image" src="https://github.com/LazyVim/LazyVim/assets/9297865/f0d653f0-7513-411e-a1af-4dddb5198fa9">

**Reply by dpetka2001** [reply]
These are just diagnostics from 2 different sources. It's normal. If you want to see diagnostics from only one source, then you have to disable the other one. That's up to you to decide.

**Reply by navxio** [reply]
I'm using the typescript and json extras and haven't defined eslint anywhere..
<img width="674" alt="image" src="https://github.com/LazyVim/LazyVim/assets/9297865/bdd9656c-ed5f-40e7-aaf2-14ca0e9b62cf">

**Reply by radvil** [reply]
can you check your mason, and verify whether eslint server was installed ?

**Reply by dpetka2001** [reply]
Do you have installed it in `Mason`? Or maybe did you install it via npm and use it with a `.eslintrc.json` file? Those diagnostics come from somewhere in your configuration.

**Reply by radvil** [reply]
I mean, just to confirm, the eslint server is not automatically installed since you first time using LazyVim.

**Reply by navxio** [reply]
I checked mason dashboard and saw that `eslint-ls` has been installed... if I remove it no hints show up at all...

**Reply by radvil** [reply]
Try remove eslint server then install typescript lang server

**Reply by navxio** [reply]
@radvil that seems to have done the trick. thank you
